library/src/main/java/com/schibsted/spain/barista/BaristaRadioButtonActions.java

```diff
@@ -13,17 +14,20 @@
 public class BaristaRadioButtonActions {
 
   public static void clickRadioButtonItem(@IdRes int radioGroupId, @IdRes int itemToClickId) {
+    scrollTo(radioGroupId);
```

Comment:
In normal clicks we just do `.perform(scrollTo(), click())`. 
Are they different? Which one is better?